internal/fs/fs.go

```diff
@@ -342,8 +343,13 @@ type fileSystem struct {
 	enableNonexistentTypeCache bool
 	inodeAttributeCacheTTL     time.Duration
 	dirTypeCacheTTL            time.Duration
-	renameDirLimit             int64
-	sequentialReadSizeMb       int32
+
+	// kernelDirCacheTTL specifies the duration to keep the dir response cached in kernel
+	// after ttl filesystem forces the kernel to evict old cache entries.
```

Comment:
nit: . After TTL, gcsfuse (filesystem) on next list (readdir) call from user asks the kernel to evict old cache entries